spec/presenters/content_items_csv_presenter_spec.rb

```diff
@@ -74,4 +74,24 @@
       ).to include('from-org-in-news-story.csv')
     end
   end
+
+  context 'when `All Organisations` is selected' do
+    subject { described_class.new(data_enum, search_params.merge(organisation_id: 'all'), document_types, organisations) }
+
+    it 'includes all-organisations in filename' do
+      expect(
+        subject.filename
+      ).to include('from-all-organisations-in-news-story.csv')
+    end
+  end
+
+  context 'when `All Organisations` is selected' do
```

Comment:
Is this meant to be a different context?
```suggestion
  context 'when `No Organisations` is selected' do
```